samples/set-max-extent/src/main/java/com/esri/arcgismaps/sample/setmaxextent/screens/SetMaxExtentScreen.kt

```diff
@@ -0,0 +1,88 @@
+/* Copyright 2025 Esri
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.esri.arcgismaps.sample.setmaxextent.screens
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.Switch
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import androidx.lifecycle.viewmodel.compose.viewModel
+import com.arcgismaps.toolkit.geoviewcompose.MapView
+import com.esri.arcgismaps.sample.setmaxextent.components.SetMaxExtentViewModel
+import com.esri.arcgismaps.sample.sampleslib.components.MessageDialog
+import com.esri.arcgismaps.sample.sampleslib.components.SampleTopAppBar
+
+/**
+ * Main screen layout for the sample app
+ */
+@Composable
+fun SetMaxExtentScreen(sampleName: String) {
+    val mapViewModel: SetMaxExtentViewModel = viewModel()
+
+    Scaffold(
+        topBar = { SampleTopAppBar(title = sampleName) },
+        content = {
+            Column(
+                modifier = Modifier
+                    .fillMaxSize()
+                    .padding(it),
+            ) {
+                MapView(
+                    modifier = Modifier
+                        .fillMaxSize()
+                        .weight(1f),
+                    arcGISMap = mapViewModel.coloradoMap,
+                    graphicsOverlays = mapViewModel.graphicsOverlays
+                )
+                Row(
+                    modifier = Modifier
+                        .fillMaxWidth()
+                        .padding(5.dp),
+                    horizontalArrangement = Arrangement.SpaceBetween,
+                ) {
+                    Text(
+                        mapViewModel.extentText.value,
+                        modifier = Modifier.align(Alignment.CenterVertically),)
+                    Switch (
+                        checked = mapViewModel.maxExtentEnabled.value,
+                        onCheckedChange = mapViewModel::onSwitch,
+                        modifier = Modifier.align(Alignment.CenterVertically),
```

Comment:
Formatting suggestion - remove the trailing commas:
```suggestion
                    horizontalArrangement = Arrangement.SpaceBetween
                ) {
                    Text(
                        mapViewModel.extentText.value,
                        modifier = Modifier.align(Alignment.CenterVertically))
                    Switch (
                        checked = mapViewModel.maxExtentEnabled.value,
                        onCheckedChange = mapViewModel::onSwitch,
                        modifier = Modifier.align(Alignment.CenterVertically)
```